modules/nw-ovn-kubernetes-ansible-migration-about.adoc

```diff
@@ -0,0 +1,145 @@
+
+. Install the `ansible-core` package, minimum version 2.15 and the `ansible-lint` package. The following example command shows how to install the `ansible-core` package on {op-system-base-full}:
++
+[source,terminal]
+----
+$ sudo dnf install -y ansible-core
+----
+
+* Install the `network.offline_migration_sdn_to_ovnk` Ansible collection by entering the following `ansible-galaxy` command:
++
+[source,terminal]
+----
+$ ansible-galaxy collection install network.offline_migration_sdn_to_ovnk
+----
+
+* Verify that the `network.offline_migration_sdn_to_ovnk` Ansible collection is installed on your system:
++
+[source,terminal]
+----
+$ ansible-galaxy collection list | grep network.offline_migration_sdn_to_ovnk 
+----
++
+.Example output
++
+[source,terminal]
+----
+# /home/<user>/.ansible/collections/ansible_collections <1>
+Collection                              Version
+------------------------                -------
+network.offline_migration_sdn_to_ovnk   2.0.0
+----
+<1> Where `<user>` is the user home directory on your system.
+
+. Disable auto-migration features by configuring the `playbooks/playbook-migration.yml` file:
++
+[source,terminal]
+----
+# ...
+    migration_disable_auto_migration: true
+    migration_egress_ip: false
+    migration_egress_firewall: false
+    migration_multicast: false
+    migration_mtu: 1400
+    migration_geneve_port: 6081
+    migration_ipv4_subnet: "100.64.0.0/16"
+# ...
+----
+where:
++
+`<migration_disable_auto_migration>`:: Disables the auto-migration of OpenShift SDN CNI plug-in features to the OVN-Kubernetes plugin. If you disable auto-migration of features, you must also set the `migration_egress_ip`, `migration_egress_firewall`, and ``migration_multicast` parameters to `false`. If you need to enable auto-migration of features, set the parameter to `false`. 
+`<migration_mtu>`:: Optional parameter that sets a specific maximum transmission unit (MTU) to your cluster network after the migration process.
+`<migration_geneve_port>`:: Optional parameter that sets a Geneve port for OVN-Kubernetes. The default port is `6081`.
+`<migration_ipv4_subnet>`:: Optional parameter that sets an IPv4 address range for internal use by OVN-Kubernetes. The default value for the OVN-Kubernetes plug-in is `100.64.0.0/16`.
+
+. After configuring the `playbook-migration.yml` file, enter the following `make` command to test the file against a `lint` rule:
++
+[source,terminal]
+----
+$ make lint
+----
+
+. Check that the `playbook-migration.yml` file has the correct dependencies by entering the following `make` command:
++
+[source,terminal]
+----
+$ make sanity
+----
+
+. Add the `playbook-migration.yml` configuration file updates to your Ansible playbook by entering the following command:
++
+[source,terminal]
+----
+$ make import
+----
+
+. Create a helper script that automates the process of authenticating with the `registry.ci.openshift.org` container image registry:
```

Comment:
@miheer Is this correct? The script does not generate the Dockerfile? If not, where is this file generated in my steps·